<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWayDialog</class>
 <widget class="QDialog" name="ChooseWayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>旅行路线</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="leastTime">
    <attribute name="title">
     <string>最短时间</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_leastTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>591</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="leastMoney">
    <attribute name="title">
     <string>最少金钱</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_leastMoney">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>591</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="limitTime">
    <attribute name="title">
     <string>限时最少金钱</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_limitTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>591</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 117, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 117, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="choosenow">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>392</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择当前策略</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
